src/client/stylesheets/index.scss

```diff
@@ -14,8 +14,8 @@
 @use 'dropdown-modifiers';
 @use 'form-control-modifiers';
 
-@import 'react-bootstrap-typeahead/css/Typeahead.css';
-@import 'react-bootstrap-typeahead/css/Typeahead.bs5.css';
+@import 'react-bootstrap-typeahead/css/Typeahead';
+@import 'react-bootstrap-typeahead/css/Typeahead.bs5';
```

Comment:
Without these changes, these imported files will be omitted from the transpiled CSS file.